Make 'hello-world' and 'two-fer' fail fast.

sshine opened the pull request:
As a follow-up to the TDD proposal of making tests fail fast in #11,
make the currently implemented exercises fail fast with this code.

This is shown to work in [this travis build](https://travis-ci.org/sshine/exercism-tcl/builds/569325997).

glennj commented:
Please see [Decide the behavior of tests that fail](https://github.com/exercism/tcl/issues/11)

sshine commented:
By the way, you can write "`#11`" to link to an issue or another PR.

I've addressed your suggestion for an environment variable.

I've addressed a bunch of other things as well.

Let me know what you think.

sshine commented on exercises/hello-world/test.tcl:
```diff
@@ -4,12 +4,33 @@ package require tcltest
 
+    set failed [failed]
     cleanupTests
+    if {$failed} {exit 1}
```
Perhaps there's a better way to do custom exiting after cleanup.

glennj replied:
Another proc? Exiting with non-zero exit status is for CI?

```tcl
proc cleanupTests {} {
    set failed [failed]
    uplevel 1 ::tcltest::cleanupTests
    if {$failed} then {exit 1}
}
```

sshine replied:
Yes, exactly. Thanks.
